habitat-lab/habitat/config/habitat/task/rearrange/multi_agent_tidy_house_base.yaml

```diff
@@ -32,29 +32,45 @@ type: RearrangeCompositeTask-v0
 reward_measure: cooperate_subgoal_reward
 success_measure: composite_success
 success_reward: 5.0
-slack_reward: -0.01
+slack_reward: -0.005
 end_on_success: True
 constraint_violation_ends_episode: False
 constraint_violation_drops_object: True
 task_spec: multi_agent_tidy_house
 actions:
   # Actions are defined per agent. Unlike the sensors, actions are not
   # automatically duplicated per agent.
+
+  # agent 0 navigation settings
   agent_0_oracle_nav_action:
     agent_index : 0
     spawn_max_dist_to_obj: -1.0
+    lin_speed: 40.0
+    ang_speed: 20.0
+  agent_0_base_velocity:
+    agent_index: 0
+    lin_speed: 40.0
+    ang_speed: 20.0
+
+  # agent 1 navigation settings
   agent_1_base_velocity:
     agent_index : 1
+    lin_speed: 40.0
+    ang_speed: 20.0
+  agent_1_oracle_nav_action:
+    agent_index : 1
+    spawn_max_dist_to_obj: -1.0
+    lin_speed: 40.0
```

Comment:
What is the difference between specifying the lin_speed here vs in base_velocity?